src/components/form/trash-list.vue

```diff
@@ -82,7 +90,13 @@ export default {
       // tell parent component (ProjectOverview) to refresh regular forms list
       // (by emitting event to that component's parent)
       this.$emit('restore');
-    }
+    },
+    onToggleTrashExpansion(evt) {
+      const projProps = this.currentUser.preferences.projects[this.project.id];
+      if (evt.newState === 'closed') {
+        projProps.formTrashCollapsed = true;
+      } else if (projProps.formTrashCollapsed) projProps.formTrashCollapsed = false;
```

Comment:
I think it'd be nice for the `else if` to parallel the `if` in terms of indentation/braces. (No braces is also totally OK, I just think it's nice for it to be consistent.)

```suggestion
      } else if (projProps.formTrashCollapsed) {
        projProps.formTrashCollapsed = false;
      }
```